mlflow/cli.py

```diff
@@ -121,6 +121,10 @@ def run(uri, entry_point, version, param_list, experiment_name, experiment_id, b
         except ValueError as e:
             eprint("Invalid cluster spec JSON. Parse error: %s" % e)
             raise
+    if backend == "kubernetes":
+        if cluster_spec_arg is None:
```

Comment:
Could we refactor all these `cluster_spec_arg`s to maybe just `backend_config`?

Reply:
done!